fastlane/Fastfile.swift

```diff
@@ -12,11 +12,36 @@ class Fastfile: LaneFile {
 
     // MARK: - Code signing
 
-    func syncCodeSigningLane() {
-        desc("This lane is for development purpose, will be removed after the migration")
-        Match.syncCodeSigning(type: .development, appIdentifier: ["co.nimblehq.ios.templates"])
-        Match.syncCodeSigning(type: .adHoc, appIdentifier: ["co.nimblehq.ios.templates"])
-        Match.syncCodeSigning(type: .appStore, appIdentifier: ["co.nimblehq.ios.templates"])
+    func syncDevelopmentCodeSigningLane() {
+        desc("Sync the development match signing for the staging build")
```

Comment:
```suggestion
        desc("Sync the Development match signing for the Staging build")
```

Reply:
Fixed in https://github.com/nimblehq/ios-templates/pull/375/commits/7e94ea9e3b9f85a503859ef4840531c83a51304c